common/lib/xmodule/xmodule/util/adaptive_learning.py

```diff
@@ -222,8 +234,12 @@ def create_knowledge_node_student(self, knowledge_node_uid, student_uid):
         """
         url = self.knowledge_node_students_url
         payload = {'knowledge_node_uid': knowledge_node_uid, 'student_uid': student_uid}
-        response = requests.post(url, headers=self.request_headers, data=payload)
-        knowledge_node_student = json.loads(response.content)
+        try:
+            knowledge_node_student = {}
```

Comment:
@macornwell This should go into the [`except` block](https://github.com/open-craft/edx-platform/pull/69#discussion_r148498599).